package.json

```diff
@@ -43,6 +43,7 @@
     "pretty-error": "^2.1.1",
     "resolve-url-loader": "^2.0.2",
     "style-loader": "^0.13.2",
+    "url-loader": "^1.0.1",
```

Comment:
should it actually be a requirement for everyone using Encore ? This is an opt-in feature.

Reply:
Yep, I hesitated a bit on that one but ended-up adding it as a main dependency without any good reason.

I moved it to dev dependencies and added the usual `ensurePackageExist` checks.